orderer/common/cluster/clusterservice.go

```diff
@@ -0,0 +1,210 @@
+/*
+Copyright IBM Corp. 2017 All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package cluster
+
+import (
+	"bytes"
+	"encoding/asn1"
+	"fmt"
+	"io"
+	"strconv"
+	"time"
+
+	"github.com/hyperledger/fabric-protos-go/orderer"
+	"github.com/hyperledger/fabric/common/flogging"
+	"github.com/hyperledger/fabric/common/util"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+//go:generate mockery -dir . -name ClusterStepStream -case underscore -output ./mocks/
```

Comment:
I think in the new mockery you need -- , not just one -, am i wrong?

Reply:
single - is supported.